plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -231,30 +236,42 @@ export class PersonState {
         }
     }
 
-    private updatedPersonProperties(personProperties: Properties): Properties {
-        const updatedProperties = { ...personProperties }
+    private updatedPersonProperties(person: Partial<Person>): Partial<Person> {
```

Comment:
The method name is misleading. Also we know what columns this is returning so please use `Pick<Person, 'foo' | 'bar'>` instead